custom_dc/env.list

```diff
@@ -0,0 +1,34 @@
+# API key for accessing the base Data Commons API.
+DC_API_KEY=
+# API key for accessing Maps and Places APIs for exploration tools.
+MAPS_API_KEY=
+# Use local SQLite as the database.
+USE_SQLITE=true
+# Input directory containing files to be processed.
+INPUT_DIR=
+# Output directory for generated csv files.
```

Comment:
Do we need to give them the option for this? Can we provide a default?

Reply:
Added a default of $INPUT_DIR/../output